app.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+
+var app = document.getElementById('app');
+var clicksRemaining = 1;
+var secondToLastPhotos = [];
+var previousPhotos = [];
+var displayedPhotos = [];
+
+
+try {
+photos = JSON.parse(localStorage.getItem('photos'));
```

Comment:
Might want to throw var in front of this, or better yet declare var photos above with your other global variables, then set it here (or below if this fails).